### DISCUSSION TITLE
Why isn’t my search function being called?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58464] [Programming Help]
### Body

My code is at https://github.com/pytutorials/pytutorials.github.io/blob/main/search.js
https://github.com/pytutorials/pytutorials.github.io/blob/main/index.md

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by emullallphotography** [comment]
Hi pytutorials, I don’t know any JavaScript but I couldn’t find any errors.

**Comment by ghost** [comment]
Then why help me??

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by emullallphotography** [reply]
To annoy you.
👍 💯  annoyed?

**Reply by emullallphotography** [reply]
Are you annoyed?

**Reply by emullallphotography** [reply]
Why aren’t you replying?

**Comment by billythekid** [comment]
Hi @pytutorials 

I've had a quick look and it appears you have an extra closing brace here:

https://github.com/pytutorials/pytutorials.github.io/blob/main/search.js#LL9C47-L9C48

Hope that helps!

Cheers,
Billy 🤠

**Reply by ghost** [reply]
Is that it?

**Reply by ghost** [reply]
It still doesn’t search for a page

**Comment by Beargruug** [comment]
you need to put the script tag for importing it before you use it like this:

```
<script src="search.js"></script>
<script src="errorhandle.js"></script>
<script src="login.js"></script>
<script src="timesVisited.js"></script>

<div class="search-form">
  <form action="javascript:search();">
    <input type="text" value="" id="search" placeholder="Search for a page..."/>
    <button onlckck="search()" id="submit">Submit</button>
    <noscript>You can't signup or search for different pages if Javascript isn't enabled.</noscript>
  </form>
</div>
<hr/>
```

hope this will help you!
